src/abilities/Stomper.js

```diff
@@ -121,27 +121,18 @@ export default (G) => {
 				else {
 					G.grid.queryDirection({
 						fnOnConfirm: function () {
-							debugger
```

Comment:
Formatted `src/abilities/Stomper.js` because it was failing linting in the pipeline. 

Removed `debugger` call, these shouldn't be left in prod code.